dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -1,16 +1,10 @@
 {% macro athena__drop_relation(relation) -%}
-  {% set rel_type = adapter.get_table_type(relation.schema, relation.table) %}
+  {% set rel_type = adapter.get_table_type(relation.schema, relation.table).value %}
```

Comment:
I am not sure to get why we need to do this 🤔

Reply:
get_table_type is returing an enum, and somehow rel_type seems to be an object not a string, but I didn't debug it
it will be good to add a debugging line to check that 
```
 {% set rel_type = adapter.get_table_type(relation.schema, relation.table)%}
```
is actually having the right value.

Reply:
Ok 👍🏻